src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { MovieType } from '../../types/Movie';
+
+type Props = {
+  movie: MovieType;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  movie: {
+    title, description, imgUrl, imdbUrl,
+  },
+}) => {
+  return (
+    <>
+      <div className="card-image">
+        <figure className="image is-4by3">
```

Comment:
It's better to destructure movie in a component
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
  const { title, description, imgUrl, imdbUrl } = movie

  return (
    <>
      <div className="card-image">
        <figure className="image is-4by3">
```